app/src/main/java/com/example/pygmyhippo/common/Entrant.java

```diff
@@ -15,7 +15,11 @@
 public class Entrant {
 
     private String accountID;
-
+    private String eventID;
+    private String name;
+    private String emailAddress;
+    private String phoneNumber;
+    private String profilePicture;
```

Comment:
is this for saving user information to an event (list of entrants)? from what i remember, did we decide on saving all the user information or just the accountID to reference in the database? i'm just thinking if the user decides to change their information, it might be hard to keep track of

Reply:
It's just used to display the information of the entrant in the list. Originally I wanted Entrant to inherit the traits from account to make it easier but just went with this instead. I think we did decide to keep it separate but it might prove more difficult to query for the matching account to get that info. The queries run asynchronously to the code so by the time it gets the results of one entrant, the code would be done setting up the array. What do you suggest?